ollama/_client.py

```diff
@@ -74,48 +74,54 @@
 T = TypeVar('T')
 
 
-class BaseClient:
+class Client:
+  @overload
```

Comment:
These overload aren't necessary since there's no overlap between the client and non-client versions

Reply:
They are to maintain proper typing, the point is you can't pass `client` and `follow_redirects` or `timeout` together. These overloads mean doing so will give a typing error.